MaxText/configs/base.yml

```diff
@@ -60,6 +60,7 @@ dtype: "bfloat16"
 quantization: ""
 # Shard the range finding operation for quantization. By default this is set to number of slices.
 quantization_local_shard_count: -1
+use_fp8: False
```

Comment:
nit: Seems like this should be quantization=fp8 two lines above? Feel free to push back.

Reply:
Okay, I can dix that.

Reply:
Done this. @rwitten @anfals